bedrock/products/templates/products/vpn/cms/resource-center/base.html

```diff
@@ -0,0 +1,25 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "products/vpn/includes/macros.html" import vpn_resource_center_cta with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}{{ page.seo_title }}{% endblock %}
```

Comment:
If we don't have `page.seo_title` set (because people may forget to add it), we should fall back to `page.title`

At the moment, we can end up with this:

![Screenshot 2024-10-08 at 15 28 24](https://github.com/user-attachments/assets/828570bd-23bf-4a3d-9d68-adecc62ff47c)

Reply:
Good suggestion! 👍